less/noscript.less

```diff
@@ -80,4 +80,38 @@ noscript {
 			}
 		}
 	}
+}
```

Comment:
Can you explain these style additions and why a `<noscript>` element can't just be used instead of a `.noscript` class? Can you also explain why these other CSS additions are necessary?